tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py

```diff
@@ -0,0 +1,2097 @@
+from constants import *
+
+CUDA_TYPE_NAME_MAP = {
```

Comment:
This file needs a description of what the format of the entries of the map are. I see some entries which are 3-tuples, and some which are 4-tuples. What are the valid values at each tuple?

Reply:
Still unaddressed

Reply:
done now - also referencing the hipify-clang source this came from